### PR TITLE
fix travis composer update

### PR DESCRIPTION
**Pull request by greg606**
Fixes #

allows travis to pass both composer install and composer update build (back to green)

## Proposed Changes

required changes include: 

- fix in `src/SWP/Bundle/CoreBundle/DependencyInjection/Compiler/OverrideBodyListenerPass` for proper argument number.
- replacing of Asm89\Twig\CacheExtension (library and namespaces) to Twig\CacheExtension

License: AGPLv3

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -53,7 +53,7 @@
     "ext-json": "*",
     "ext-zip": "*",
     "apache/log4php": "^2.3",
-    "asm89/twig-cache-extension": "^1.4",
+    "twig/cache-extension": "^1.4",
```
Comment (ahilles107) [inline composer.json:56]: 👍

**Comment by ahilles107** [comment]
Is this ready fir merge? What's the status of it?

**Comment by greg606** [comment]
> Is this ready fir merge? What's the status of it?

yes, I think so.

**Comment by greg606** [comment]
@ahilles107 Do you think we can already merge it?

**Comment by ahilles107** [comment]
Yes :)

**Pull request merged**